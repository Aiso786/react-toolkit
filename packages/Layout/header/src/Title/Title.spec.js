import React from 'react';
import renderer from 'react-test-renderer';
import { Title } from './index';

describe('<Title>', () => {
  it('renders Title correctly', () => {
    const tree = renderer
      .create(
        <Title
          title="Toolkit Axa"
          subtitle="Info complémentaire"
          toggleMenu={() => null}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders Title correctly without menu', () => {
    const tree = renderer
      .create(<Title title="Toolkit Axa" subtitle="Info complémentaire" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
