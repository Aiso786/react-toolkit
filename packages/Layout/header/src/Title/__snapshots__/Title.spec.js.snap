// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title> renders Title correctly 1`] = `
<div
  className="af-title-bar"
>
  <div
    className="container af-title-bar__wrapper"
  >
    <div
      className="burger-container"
    >
      <a
        aria-controls="mainmenu"
        aria-haspopup={true}
        className="btn af-title-bar__mobile-menu af-btn--circle"
        href="#"
        id="togglemenu"
        onClick={[Function]}
        role="button"
        tabIndex={0}
        title="Toggle menu"
      >
        <i
          className="glyphicon glyphicon-menu-hamburger"
        />
      </a>
    </div>
    <h1
      className="af-title-bar__title"
    >
      Toolkit Axa
      <small
        className="af-title-bar__subtitle"
      >
        Info complémentaire
      </small>
    </h1>
  </div>
</div>
`;

exports[`<Title> renders Title correctly without menu 1`] = `
<div
  className="af-title-bar"
>
  <div
    className="container af-title-bar__wrapper"
  >
    <h1
      className="af-title-bar__title"
    >
      Toolkit Axa
      <small
        className="af-title-bar__subtitle"
      >
        Info complémentaire
      </small>
    </h1>
  </div>
</div>
`;
